ag/ag.proto

```diff
@@ -65,22 +65,23 @@ message Groups {
 //   COURSES   //
 
 message Course {
-    uint64 ID                      = 1;
-    uint64 courseCreatorID         = 2;
-    string name                    = 3;
-    string code                    = 4 [(go.field) = { tags: 'gorm:"uniqueIndex:idx_unique_course"' }];
-    uint32 year                    = 5 [(go.field) = { tags: 'gorm:"uniqueIndex:idx_unique_course"' }];
-    string tag                     = 6;
-    string provider                = 7;
-    uint64 organizationID          = 8;
-    string organizationPath        = 9;  // The organization's SCM name, e.g. uis-dat520-2020.
-    uint32 slipDays                = 10;
-    string dockerfile              = 11;
-    Enrollment.UserStatus enrolled = 12 [(go.field) = { tags: 'gorm:"-"' }];
-
-    repeated Enrollment enrollments = 13;
-    repeated Assignment assignments = 14;
-    repeated Group groups           = 15;
+    uint64 ID = 1;
```

Comment:
Could you install `clang-format` in `/usr/bin`? (see the `.vscode/settings.json`) This would allow you to format the `ag.proto` the same as me, avoiding that the field numbers gets reflowed...